payment/controllers_test.go

```diff
@@ -472,40 +499,13 @@ func (suite *ControllersTestSuite) AnonymousCardTestE2E() {
 	handler.ServeHTTP(rr, req)
 	suite.Assert().Equal(http.StatusOK, rr.Code)
 
-	handler = GetOrderCreds(service)
```

Comment:
Since this is an end to end test I think we should either extract this into a function to test or put this back in

Reply:
in latest commit.